Apps/WebClient/src/ClientApp/src/components/report/medicationRequest.vue

```diff
@@ -90,17 +115,55 @@ export default class MedicationRequestReportComponent extends Vue {
         );
     }
 
-    public async generatePdf(): Promise<void> {
+    public generatePdf(): Promise<void> {
```

Comment:
Was `async` supposed removed from the `generatePdf()` function in this file but not the ones in the other files?

Reply:
No, it wasn't, although AFAIK it does not make much difference since both return a promise. I have added it back